brave/brave-core
# Restoring wallet can now be done from file

Fixes brave/brave-browser#1325

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Open Brave and go to chrome://rewards.
2. Try to restore a wallet backup from a wallet backup file.
3. Ensure that wallet is restored properly.

-

1. Open Brave and go to chrome://rewards.
2. Try to restore a wallet backup by pasting backup words.
3. Ensure that wallet is restored properly and pasting words did not regress.


## Reviewer Checklist:
